lib/fastlane_core/command_executor.rb

```diff
@@ -59,6 +59,9 @@ def execute(command: nil, print_all: false, print_command: true, error: nil, pre
             end
             Process.wait(pid)
           end
+        rescue Errno::EIO
+          # This is expected on some linux systems, that indicates that the subcommand finished
+          # and we kept trying to read, ignore it
```

Comment:
In this case the only difference looks like the `Process.wait(pid)` line.  I think this is easier to read combined as a special case of the other rescue, and in the future is probably the safer place to catch that error, anything else in the block would need to stop when the process dies.